tests/examples/shims/mouse.lua

```diff
@@ -53,20 +53,28 @@ function mouse.push_history()
     mouse.history = {}
 end
 
+local forced_screen = nil
+
 return setmetatable(mouse, {
     __index = function(self, key)
         if key == "screen" then
+            if forced_screen and screen._deleted[forced_screen] then
```

Comment:
What's the difference between `screen._deleted[s]` and `s.valid`? I would have expected that this patch assumes that no one "messes with" `s.valid` and just add `assert(s.valid)` in some places...?

Reply:
There's a slim way to get a client (with its own `.valid`) interpreted as a screen if someone adds an `outputs` property. The `_deleted` table is harder to corrupt.